src/main/java/com/skankhunt220/configuration/ApplicationConfig.java

```diff
@@ -0,0 +1,14 @@
+package com.skankhunt220.configuration;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.mongodb.config.AbstractMongoClientConfiguration;
+import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;
+
+@Configuration
+@EnableMongoRepositories(basePackages = "com.skankhunt220.repository")
+class ApplicationConfig extends AbstractMongoClientConfiguration {
+	@Override
+	protected String getDatabaseName() {
```

Comment:
You can create a application.property file and Spring autoconfigure your mongoDb(this file can be place in **src/main/resources**)
Example file:
```
mongodb.database=test
mongodb.host=localhost
mongodb.port=27017
```
and etc.